README.md

```diff
@@ -173,6 +173,95 @@ crud.insert_object('customers', {
 ...
 ```
 
+### Insert many
+
+```lua
+-- Insert batch of tuples
+local result, err = crud.insert_many(space_name, tuples, opts)
+-- Insert batch of objects
+local result, err = crud.insert_object_many(space_name, objects, opts)
+```
+
+where:
+
+* `space_name` (`string`) - name of the space to insert an object
+* `tuples` / `objects` (`table`) - array of tuples/objects to insert
+* `opts`:
+  * `timeout` (`?number`) - `vshard.call` timeout (in seconds)
+  * `fields` (`?table`) - field names for getting only a subset of fields
+  * `stop_on_error` (`?boolean`) - stop on a first error and report errors
+    regarding the failed operation and all not performed ones, default is
+    `false`
+  * `rollback_on_error` (`?boolean`) - any failed operation will lead to
+    rollback on a storage, where the operation is failed, default is
+    `false`
+
+Returns metadata and array contains inserted rows, array of errors
+(one error corresponds to one replicaset for which the error occurred
+if `stop_on_error` is `true`).
```

Comment:
I updated this section:

```Returns metadata and array contains inserted rows, array of errors.
Error object can contain field `operation_data` field.

This field can contain:
* tuple for which the error occurred;
* object with an incorrect format;
* tuples that were rollback;
* tuples that were not performed
  because operation was stopped by error.
```

Reply:
We discussed it and agreed to leave one object/tuple in the `operation_data` field. If several tuples are rolled back, just add several errors. The API anyway assumes returning several errors, so it would be quite natural (and easier to handle).